comp/core/flare/component.go

```diff
@@ -10,17 +10,18 @@
 package flare
 
 import (
+	"go.uber.org/fx"
+
 	"github.com/DataDog/datadog-agent/comp/core/flare/helpers"
 	"github.com/DataDog/datadog-agent/pkg/util/fxutil"
-	"go.uber.org/fx"
 )
 
 // team: agent-shared-components
 
 // Component is the component type.
 type Component interface {
 	// Create creates a new flare locally and returns the path to the flare file.
-	Create(pdata ProfileData, ipcError error) (string, error)
+	Create(pdata ProfileData, providerTimeoutSeconds int, ipcError error) (string, error)
```

Comment:
The only reason is that both the CLI flag and the config value are ints (that's simpler for users), so taking an int as parameter allows doing the conversion to `Duration` once in the function, rather than at each call site
If you think it's better to take a `Duration` directly then let me know I can update it

Reply:
It's true that if we already have one duration arg then we might as well be coherent and use a duration for that one too 🤔

Reply:
Ok that convinced me, I'll update the code 😁 👍